test/fixtures/unicorn/prefer-text-content.pass.ts

```diff
@@ -0,0 +1,3 @@
+/**
+ * TODO: this is for browser, find good way to test.
```

Comment:
JSDom?

Reply:
left an expiring todo for this in https://github.com/ridedott/eslint-config/pull/319/commits/09dc2a5dc7793d9447eb5b3402b77e064cece216